ocean_lib/ocean/ocean_assets.py

```diff
@@ -230,6 +230,10 @@ def create(
             if owner_address:
                 data_token.proposeMinter(owner_address, from_wallet=publisher_wallet)
         else:
+            if not dtfactory.verify_data_token(data_token_address):
```

Comment:
When I have implemented unit tests for the exceptions from `exceptions.py`, I have noticed here that we didn't verify in the first place if the contract actually exists. And I have switched some conditions here, because if we want to find the `minter`, firstly we need to check if the contract exists. What are your thoughts here @trentmc? 
P.S. : Don't want to break the interface again :(

Reply:
I think this is good, but I defer to @trentmc.

Reply:
For a given unit test that uses a contract, it will fail if the contract doesn't exist. This is fine. You don't need to make it more explicit.

FYI dozens of unit tests assume the contract exists. If the contracts don't exist, dozens of unit tests will fail. This is fine.